python/dgl/dataloading/pytorch/__init__.py

```diff
@@ -13,6 +13,129 @@ def _remove_kwargs_dist(kwargs):
         print('Distributed DataLoader does not support pin_memory')
     return kwargs
 
+# The following code is a fix to the PyTorch-specific issue in
+# https://github.com/dmlc/dgl/issues/2137
+#
```

Comment:
...

The sampled blocks/subgraphs contain features extracted from the parent graph. In DGL, the sub-feature
data structure holds a reference to the parent graph feature tensor and an index tensor. When transmitting
the sampled block/subgraph to the main process, it is unnecessary to transmit the parent graph feature
back.

To achieve that, in the collator ....

Reply:
K.